packages/components/organisms/f-content-cards/README.md

```diff
@@ -13,13 +13,83 @@
 [![Coverage Status](https://coveralls.io/repos/github/justeat/f-content-cards/badge.svg)](https://coveralls.io/github/justeat/f-content-cards)
 [![Known Vulnerabilities](https://snyk.io/test/github/justeat/f-content-cards/badge.svg?targetFile=package.json)](https://snyk.io/test/github/justeat/f-content-cards?targetFile=package.json)
 
-## Prerequisites
+# Content Cards (f-content-cards)
+
+## Introduction
+
+The content cards component is a headless vue component that takes an array of adapters (functions that return cards) and exposes the cards collected from these adapters via the slot prop `cards`. The component also handles the tracking events for viewing and clicking cards, which are then sent back through to the relevant adapter.
+
+## Structure
+
+The component has the following folder structure
+
+- cards (folder for new cards to be placed in)
+- cardTemplates (deprecated folder for older cards not yet updated)
+- templates (components which are sections of content cards, used to make cards)
+- ContentCards.js (Main headless component)
+
+## Main Component (ContentCards.js)
+
+### Props
+
+| Name     | Type   | Required | Default       | Purpose                             |
+|----------|--------|----------|---------------|-------------------------------------|
+| adapters | Array  | true     | N/A           | Functions that return content cards |
```

Comment:
Based on [here](https://github.com/justeattakeaway/fozzie-components/pull/2233/files#diff-a04a7ac3633f5f9f393e9cfbbfba8ba14be8accc39168abd2feaa6b18cc84865R25-R38), the description for this element isn't exactly true. I'd see this as something more like:

> A list of objects that conform to the 'Adapter' interface.

You could then list out what is involved for an adapter interface in more detail in an `Adapters` section.

Reply:
very true